draft-bbs-signatures.md

```diff
@@ -428,7 +428,7 @@ Procedure:
 
 6. for element in (e, s) do
 
-7.      element = OS2IP(h.read(xof_no_of_bytes)) mod q
+7.      element = OS2IP(h.read(xof\_no\_of\_bytes)) mod q
```

Comment:
I dont think these need to be escaped do they? just the ones outside of a code block?

Reply:
Good catch. Fixed (here and in CreateGenerators)